src/arrays.js

```diff
@@ -1,61 +1,157 @@
 const getNthElement = (index, array) => {
-  // your code here
+  const maxArrayindex = (array.length - 1);
+  // console.log(maxArrayindex);
+  const indexGiven = index;
```

Comment:
what's the purpose of this new constant? Can you simply not use `index` argument provided?